bundle/manifests/operator.clusterserviceversion.yaml

```diff
@@ -21,10 +21,6 @@ metadata:
                       }
                   },
                   "fluentd": {
-                      "buffer": {
-                          "queueChunkLimitSize": 256,
-                          "totalLimitSize": "256m"
-                      },
```

Comment:
Why is this change necessary?

Reply:
This parameters are set for OpenShift in e2e tests but they are not mentioned in our documentation for sumologic helm chart. I see that collection works without these parameters set - so I'm removing them from example configuration.